esmvaltool/recipes/schlund20jgr/recipe_schlund20jgr_gpp_change_rcp85.yml

```diff
@@ -0,0 +1,947 @@
+# ESMValTool
+# recipe_schlund20jgr_gpp_change_rcp85.yml
+---
+documentation:
+
+  description: |
+    Use MLR (Machine Learning Regression) models to contstrain fractional
+    change in projected gross primary production (GPP) in the emission-driven
+    RCP 8.5 scenario.
+
+  authors:
+    - schlund_manuel
+
+  maintainer:
+    - schlund_manuel
+
+  references:
+    - schlund20jgr
+
+  projects:
+    - 4c
+    - crescendo
+    - usmile
+
+  domains:
+    - global
+
+  realms:
+    - land
+
+  themes:
+    - ML
+    - EC
+    - carbon
```

Comment:
I'm not sure if it works if you specify this here, typically `themes` and `realms` are specified in the diagnostic sections and `domains` is something that can be specified in the provenance record coming from the diagnostic. See also the documentation on recording provenance: https://docs.esmvaltool.org/en/latest/community/diagnostic.html#recording-provenance

Reply:
Ah, I didn't know that. Would it make sense to implement that? I think we should be very flexible here (allow setting all provenance options in the recipe, diagnostic section and diagnostic itself). In this case it would make sense to set these option in the recipe for all diagnostic. Setting them to fixed values in the diagnostic script itself makes no sense since it works on arbitrary input variables (type and shapes).

Reply:
This is how it's currently implemented in the ESMValCore, so for now I think it is best to implement that here. Maybe you could open an issue in the ESMValCore repository to propose to make this more flexible?

Reply:
So you mean adding this to every diagnostic? This won't look nice, but I can do it. I will open an issue about that.

Reply:
> So you mean adding this to every diagnostic?

Yes, because then it works, see e.g. examples/recipe_python.yml or recipe_perfmetrics_CMIP5.yml for how it's done there